# id of object, must be unique
int32 object_id

# name of object
string object_name

# type: "edge" or "vertex"
string object_type

# for edges: start and end vertices
# for vertices: not used
int32[] object_references

